--- third_party/WebKit/Source/modules/webdatabase/Database.h.orig
+++ third_party/WebKit/Source/modules/webdatabase/Database.h
@@ -28,6 +28,7 @@
 
 #include "modules/webdatabase/DatabaseBasicTypes.h"
 #include "modules/webdatabase/DatabaseError.h"
+#include "modules/webdatabase/SQLTransactionBackend.h"
 #include "modules/webdatabase/sqlite/SQLiteDatabase.h"
 #include "platform/bindings/ScriptWrappable.h"
 #include "platform/bindings/TraceWrapperMember.h"
