--- third_party/WebKit/Source/modules/webdatabase/SQLTransactionBackend.h.orig
+++ third_party/WebKit/Source/modules/webdatabase/SQLTransactionBackend.h
@@ -31,6 +31,7 @@
 #include <memory>
 #include "modules/webdatabase/DatabaseBasicTypes.h"
 #include "modules/webdatabase/SQLStatement.h"
+#include "modules/webdatabase/SQLStatementBackend.h"
 #include "modules/webdatabase/SQLTransactionStateMachine.h"
 #include "platform/heap/Handle.h"
 #include "platform/wtf/Deque.h"
