diff -up chromium-61.0.3163.79/third_party/WebKit/Source/platform/wtf/typed_arrays/ArrayBufferContents.h.gcc5fix chromium-61.0.3163.79/third_party/WebKit/Source/platform/wtf/typed_arrays/ArrayBufferContents.h
--- third_party/WebKit/Source/platform/wtf/typed_arrays/ArrayBufferContents.h.gcc5fix	2017-09-06 16:33:35.213611900 -0400
+++ third_party/WebKit/Source/platform/wtf/typed_arrays/ArrayBufferContents.h	2017-09-06 16:34:34.471469606 -0400
@@ -63,7 +63,7 @@ class WTF_EXPORT ArrayBufferContents {
           allocation_length_(0),
           data_(data),
           data_length_(0),
-          kind_(AllocationKind::kNormal),
+          kind_(WTF::ArrayBufferContents::AllocationKind::kNormal),
           deleter_(deleter) {}
     DataHandle(void* allocation_base,
                size_t allocation_length,
@@ -94,11 +94,11 @@ class WTF_EXPORT ArrayBufferContents {
              reinterpret_cast<uintptr_t>(allocation_base_) +
                  allocation_length_);
       switch (kind_) {
-        case AllocationKind::kNormal:
+        case WTF::ArrayBufferContents::AllocationKind::kNormal:
           DCHECK(deleter_);
           deleter_(data_);
           return;
-        case AllocationKind::kReservation:
+        case WTF::ArrayBufferContents::AllocationKind::kReservation:
           ReleaseReservedMemory(allocation_base_, allocation_length_);
           return;
       }
diff -up chromium-61.0.3163.79/third_party/webrtc/modules/audio_processing/aec3/aec_state.cc.gcc5fix chromium-61.0.3163.79/third_party/webrtc/modules/audio_processing/aec3/aec_state.cc
--- third_party/webrtc/modules/audio_processing/aec3/aec_state.cc.gcc5fix	2017-09-06 16:34:51.959131969 -0400
+++ third_party/webrtc/modules/audio_processing/aec3/aec_state.cc	2017-09-06 16:35:03.027919336 -0400
@@ -10,7 +10,7 @@
 
 #include "webrtc/modules/audio_processing/aec3/aec_state.h"
 
-#include <math.h>
+#include <cmath>
 #include <numeric>
 #include <vector>
 
