--- third_party/angle/third_party/vulkan-validation-layers/BUILD.gn.orig
+++ third_party/angle/third_party/vulkan-validation-layers/BUILD.gn
@@ -273,8 +273,8 @@
     cflags = [ "/wd4201" ]
   }
   if (is_linux) {
-    # assume secure_getenv() is available
-    defines += [ "HAVE_SECURE_GETENV" ]
+    # assume getauxval() is available
+    defines += [ "HAVE_GETAUXVAL" ]
   }
 }
 
--- third_party/angle/third_party/vulkan-validation-layers/src/loader/loader.c.orig
+++ third_party/angle/third_party/vulkan-validation-layers/src/loader/loader.c
@@ -31,6 +31,9 @@
 #include <stdbool.h>
 #include <string.h>
 #include <stddef.h>
+#if defined(HAVE_GETAUXVAL)
+#include <sys/auxv.h>
+#endif
 
 #include <sys/types.h>
 #if defined(_WIN32)
@@ -222,6 +225,8 @@
     return secure_getenv(name);
 #elif defined(HAVE___SECURE_GETENV)
     return __secure_getenv(name);
+#elif defined(HAVE_GETAUXVAL)
+    return getauxval(AT_SECURE) ? NULL : getenv(name);
 #else
 #pragma message(                                                                       \
     "Warning:  Falling back to non-secure getenv for environmental lookups!  Consider" \
